.github/workflows/build.yml

```diff
@@ -35,21 +35,14 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
+        cache: 'maven'
 
     - name: Get Date
```

Comment:
if applying the get date section should go too.

Reply:
Exactly. I hadn't decided to apply yet. 
Flipping to the cache function provided by the GH Action could make sense.. but it would change the cache refresh frequency. I don't know how to observe behavior to measure/understand the impact of this change (which is why I asked in Zulip).

Reply:
I think it should be fine for a Quarkiverse extension but I would remove the artifacts installed by this build from the cache at the end of the build (so before the caching that will happen at the end of the build). Otherwise the cache will always be stale.